crates/tuono_lib/tests/utils/form_data.rs

```diff
@@ -1,13 +1,12 @@
+use axum::extract::Form;
 use serde::Deserialize;
-use tuono_lib::Request;
 
 #[derive(Deserialize)]
-struct Payload {
+pub struct Payload {
```

Comment:
does it need to be `pub`?

Reply:
They could be `pub(crate)`, but since they're test files I didn't worry about it. They need to be visible to the mock server or you get 

```
error: type `form_data::Payload` is private
  --> crates/tuono_lib/tests/utils/mock_server.rs:90:43
   |
90 |             .route("/api/form_data", post(form_data_api))
   |                                           ^^^^^^^^^^^^^ private type
```